x-pack/plugins/observability_solution/synthetics/server/routes/default_alerts/default_alert_service.test.ts

```diff
@@ -0,0 +1,466 @@
+      setupTlsRule.mockResolvedValueOnce({ status: 'fulfilled', value: {} });
+      const result = await service.setupDefaultAlerts();
+      expect(setupStatusRule).toHaveBeenCalledTimes(1);
+      expect(setupTlsRule).toHaveBeenCalledTimes(1);
+      expect(result).toEqual({
+        statusRule: { status: 'fulfilled', value: {} },
+        tlsRule: { status: 'fulfilled', value: {} },
+      });
+    });
+    it('returns null rules if value is falsy', async () => {
+      const soClient = { get: jest.fn() } as any;
+      const service = new DefaultAlertService({} as any, {} as any, soClient);
+      service.getSettings = jest.fn().mockResolvedValue({
+        certAgeThreshold: 50,
+        certExpirationThreshold: 10,
+        defaultConnectors: ['slack', 'email'],
+        defaultEmail: undefined,
+        defaultStatusRuleEnabled: true,
+        defaultTLSRuleEnabled: true,
+      });
+      const setupStatusRule = jest.fn();
+      const setupTlsRule = jest.fn();
+      service.setupStatusRule = setupStatusRule;
+      service.setupTlsRule = setupTlsRule;
+      setupStatusRule.mockResolvedValueOnce(undefined);
+      setupTlsRule.mockResolvedValueOnce(undefined);
+      const result = await service.setupDefaultAlerts();
+      expect(setupStatusRule).toHaveBeenCalledTimes(1);
+      expect(setupTlsRule).toHaveBeenCalledTimes(1);
+      expect(result).toEqual({
+        statusRule: null,
+        tlsRule: null,
+      });
+    });
+  });
+
+  describe('getMinimumRuleInterval', () => {
+    it('returns 1m if minimum interval is less than 1m', () => {
+      const server = {
+        alerting: { getConfig: () => ({ minimumScheduleInterval: { value: '30s' } }) },
+      } as any;
+      const service = new DefaultAlertService({} as any, server, {} as any);
+      expect(service.getMinimumRuleInterval()).toBe('1m');
+    });
+
+    it('returns minimum interval if greater than 1m', () => {
+      const server = {
+        alerting: { getConfig: () => ({ minimumScheduleInterval: { value: '5m' } }) },
+      } as any;
+      const service = new DefaultAlertService({} as any, server, {} as any);
+      expect(service.getMinimumRuleInterval()).toBe('5m');
+    });
+  });
+
+  describe('setupStatusRule', () => {
+    it('creates status rule if enabled', async () => {
+      const service = new DefaultAlertService({} as any, {} as any, {} as any);
+      service.getMinimumRuleInterval = jest.fn().mockReturnValue('1m');
+      service.createDefaultAlertIfNotExist = jest.fn();
+      service.settings = { defaultStatusRuleEnabled: true } as any;
+      service.getSettings = jest.fn().mockResolvedValue({
+        defaultStatusRuleEnabled: true,
+      });
+      await service.setupStatusRule();
+      expect(service.createDefaultAlertIfNotExist).toHaveBeenCalledWith(
+        SYNTHETICS_STATUS_RULE,
+        'Synthetics status internal rule',
+        '1m'
+      );
+    });
+
+    it('does not create status rule if disabled', async () => {
```

Comment:
We can leave this here, but this is one that will probably be removed next cycle as we remove the settings for enabling and disable rules.

Reply:
Yeah as I wrote a few of these I was like... testing features that do not exist... but they were pretty fast to write and it will give validation when we remove them that the module is tested at least.